tests/monday_type_tests.py

```diff
@@ -84,3 +84,74 @@ def test_should_succeed_when_to_primitive_returns_str_when_passed_in_an_int_floa
     eq_(int_value,'1')
     eq_(float_value,'1.0')
     eq_(str_value,'text')
+
+
+def test_should_succeed_when_to_native_returns_a_str_when_passing_a_longtext_value_with_api_data():
```

Comment:
Similar to before, please put the type that you are testing in the name of the test method.

Ex. test_longtext_type_should_succeed_when_...

Reply:
Ok